db/migrate/20150808115436_remove_requirement_from_users_invitation_code.rb

```diff
@@ -0,0 +1,5 @@
+class RemoveRequirementFromUsersInvitationCode < ActiveRecord::Migration
+  def change
+    change_column_null :users, :invitation_code, true, ENV['INVITATION_CODE']
```

Comment:
I don't know if we want to rely on an ENV variable to be set correctly at migration time.  It also could change later.  Maybe it'd be better to drop the column and have the invitation code check only run on create?